File: trunk/net.tenteCsharp/src_code/heaterManagement/IGeneralHeaterNotify.java
package heaterManagement;
import heaterManagement.HeaterManagement.*;

public interface IGeneralHeaterNotify{
	
	public void switchOnAllHeaters(String floorId,String roomId);	
	public void switchOffAllHeaters(String floorId,String roomId);	
	public void changeAllHeatersTemp(float temperature,String floorId,String roomId);

}

File: trunk/pseudoCodeGeneratorElio/src-gen/heaterManagement/IGeneralHeaterNotify.cs
using System;
using System.Collections;
using System.Text;

namespace SmartHome
{
	public interface IGeneralHeaterNotify
	{
		void switchOnAllHeaters(String floorId, String roomId);
		void switchOffAllHeaters(String floorId, String roomId);
		void changeAllHeatersTemp(float temperature, String floorId, String roomId);

	}
}
